src/server/report/app.js

```diff
@@ -160,20 +155,15 @@ class App extends LitElement {
 	}
 	render() {
 		return html`
-			<div class="container">
-				<aside>
-					<div>
-						<h1>Visual-diff Results</h1>
-						${this._renderFilters()}
-					</div>
-				</aside>
-				<main>${this._renderMainView()}</main>
-			</div>
+			<aside>
+				<div>
+					<h1>Visual-diff Results</h1>
+					${this._renderFilters()}
+				</div>
+			</aside>
+			<main>${this._renderMainView()}</main>
 		`;
 	}
-	_handleBackClick() {
-		this._updateSearchParams({ file: undefined, test: undefined });
-	}
```

Comment:
This and `_handleListTestClick` didn't end up being necessary due to the `<a href>` doing the same thing -- it was actually firing things twice.